modules/apps/portal/portal-upgrade-test-util/src/main/java/com/liferay/portal/upgrade/test/util/BaseExternalReferenceCodeUpgradeProcessTestCase.java

```diff
@@ -110,6 +110,27 @@ public void testUpgradeProcess() throws Exception {
 			addExternalReferenceCodeModels(String tableName)
 		throws PortalException;
 
+	protected void assertExternalReferenceCode(
```

Comment:
why did you change this class?

Reply:
because I need to override some method because in the role table there is no the groupId column (that is checked in the base one)